internal/show/brokers.go

```diff
@@ -51,14 +49,20 @@ func Brokers(clusterInfo *cluster.Info, status reporter.Interface) bool {
 		return true
 	}
 
-	fmt.Printf(template, "NAMESPACE", "NAME", "COMPONENTS")
+	template := "%-28s%-25.24s%-38s%-24v%-22s%-30v%-60s\n"
```

Comment:
```suggestion
	template := "%-25.24s%-25.24s%-34.33s%-20v%-21s%-26v%-40s\n"
```

I'd use a more compact scheme, resulting in 192 character long print (instead of 228):
```
NAMESPACE                NAME                     COMPONENTS                        GLOBALNET ENABLED   GLOBALNET CIDR       DEFAULT GLOBALNET SIZE    DEFAULT DOMAINS                         
submariner-k8s-broker    submariner-broker        connectivity, service-discovery   false               242.0.0.0/8          65536                                                             
```